src/include/cp_action.h

```diff
@@ -188,16 +188,16 @@ struct __wt_control_point_pair_data_err {
 
 /* Macro to define a per-connection control point with this action. */
 #ifdef HAVE_CONTROL_POINT
-#define CONNECTION_CONTROL_POINT_DEFINE_ERR(SESSION, CONTROL_POINT_ID)             \
-    CONNECTION_CONTROL_POINT_DEFINE_START((SESSION), (CONTROL_POINT_ID), false, 0) \
-    if (_cp_data != NULL) {                                                        \
-        int _err = ((WT_CONTROL_POINT_PAIR_DATA_ERR *)_cp_data)->err;              \
-        /* _cp_data not needed during action. */                                   \
-        __wt_control_point_release_data(_session, _cp_registry, _cp_data, false);  \
-        _cp_data = NULL;                                                           \
-        /* The action. */                                                          \
-        WT_ERR(_err);                                                              \
-    }                                                                              \
+#define CONNECTION_CONTROL_POINT_DEFINE_ERR(SESSION, CONTROL_POINT_ID)            \
+    CONNECTION_CONTROL_POINT_DEFINE_START((SESSION), (CONTROL_POINT_ID))          \
+    if (_cp_data != NULL) {                                                       \
```

Comment:
These are some existing errors I found.

Change new line 133 from
uint64_t _seconds = action_data->sleep;  \
to
 uint64_t _seconds = action_data->seconds; \

Change new line 252 from
 __wt_control_point_release_data(_session, _cp_register, _cp_data, false);  \
to
  __wt_control_point_release_data(_session, _cp_registry, _cp_data, false);  \